src/Microsoft.TestPlatform.Common/Hosting/ITestRuntimeProviderManager.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.VisualStudio.TestPlatform.ObjectModel.Host;
```

Comment:
header, usings, namespace, seemed most natural to me, and is also what the "move type to file" refactoring uses. But it is not what the current rules mandated. Personally I don't really care which one it will be. But I was fighting a bit to getting the build to pass because of all these issues that are not identified at first, and then in later builds they appear. It is not random, but I also don't feel like I am getting all warnings all the time, as you noticed.

Reply:
> header, usings, namespace, seemed most natural to me

Will do a style change update later then.

> is also what the "move type to file" refactoring uses

On my side, the refactoring seems to be correctly following setup (either inside or outside).

> It is not random, but I also don't feel like I am getting all warnings all the time, as you noticed.

Same problem here.